portfolio/src/main/java/com/google/sps/servlets/TitanicDataServlet.java

```diff
@@ -36,22 +37,39 @@ public void init() {
       String line = scanner.nextLine();
       String[] cells = line.split(",");
 
-      Integer passengerSurvived = Integer.valueOf(cells[FIRST_COLUMN_SURVIVED]);
+      while (cells.length != 2) {
```

Comment:
Instead of using a separate loop for this, using an `if` statement and the the `continue;` keyword would be more simple, and reduces code duplication.